pt-BR/mail/chrome/messenger/subscribe.properties

```diff
@@ -35,9 +35,9 @@
 #
 # ***** END LICENSE BLOCK *****
 
-subscribeLabel-nntp=Inscrever os newsgroups selecionados:
+subscribeLabel-nntp=Inscrever os grupos de notícias selecionados:
 subscribeLabel-imap=Inscrever as pastas selecionadas:
-currentListTab-nntp.label=Current Group List
+currentListTab-nntp.label=Lista de grupos atual
 currentListTab-nntp.accesskey=L
 currentListTab-imap.label=Listas de pastas
```

Comment:
CORRECTIONS NEEDED it should be currentListTab-imap.label=Listas de pastas atual